common/utils/start.sh

```diff
@@ -41,7 +41,22 @@ then
     fi
 fi
 
-# Set up the env for the spark user
+# Determine SPARK_CONF_DIR. If a non-empty config has been given then use it
+ls -1 /etc/oshinko-spark-configs &> /dev/null
```

Comment:
the path here can be set from the templates in an environment variable to make it a little more loosely coupled